src/vs/workbench/contrib/chat/electron-browser/tools/fetchPageTool.ts

```diff
@@ -278,28 +278,31 @@ export class FetchWebPageTool implements IToolImpl {
 		return { webUris, fileUris, invalidUris };
 	}
 
-	private _getPromptPartsForResults(results: ResultType[]): (IToolResultTextPart | IToolResultDataPart)[] {
-		return results.map(value => {
+	private _getPromptPartsForResults(urls: string[], results: ResultType[]): (IToolResultTextPart | IToolResultDataPart)[] {
+		return results.map((value, i) => {
+			const title = results.length > 1 ? localize('fetchWebPage.fetchedFrom', 'Fetched from {0}', urls[i]) : undefined;
```

Comment:
I suggest we wrap the URL in backticks "`" and support making the title a `string | IMarkdownString` to make the display a little nicer